### DISCUSSION TITLE
Who Is the creator of react-router?

### DISCUSSION THREAD

**Discussion by sidx8** [remix-run/react-router#8606] [Q&A] [answered]
Couldn't find much information about this but the oldest commit is by @timdorr so I'm assuming it's him can somebody pls confirm it I am making a GitHub explore topic about react-router

**Comment by timdorr** [comment, marked as answer]
The first commit was by @ryanflorence: 987de78deb9687f15133188f2e8e51ffd653794d

The library is co-created by @ryanflorence and @mjackson. I'm just a long-time contributor.

**Reply by sidx8** [reply]
thanks and appreciate your contributions to the project

**Reply by MeiKatz** [reply]
Okay, note for everybody: 4th May 2024 is the ten year anniversary of the first RR commit. #SaveTheDate
